docs/core/diagnostics/index.md

```diff
@@ -21,7 +21,9 @@ This article helps you find the various tools you need.
 
 ## Metrics
 
-[EventCounters](event-counters.md) allows you to write metrics to identify and monitor performance issues. Metrics incur lower performance overhead compared to tracing, making it more suitable for an always-on performance monitoring. The .NET runtime and libraries publish several [well-known EventCounters](available-counters.md) that you can monitor as well.
+[Metrics](metrics.md) are numerical measurements recorded by apps and libraries over time, often to monitor application health and generate alerts when potential
```

Comment:
Something about mixing "app" and "application" in the same sentence seems funny to me.

Here's my stab at wordsmithing it:

Metrics are numerical measurements taken while code is running and analyzed over time to monitor the health and/or performance of an application or system. Metrics are often used to generate alerts when potential problems are detected.

🤷

Reply:
I did some more tweaking and got here. How does this sound to you?

`Metrics are numerical measurements recorded over time to monitor application performance and health. Metrics are often used to generate alerts when potential problems are detected.`